### PR TITLE
components - Add calculator

### PR DESCRIPTION
**Pull request by codecaiine**
- Added new calculator component
- Added buttons for different numbers and operation
- Added Screen to display information
- Fixed linters errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
+  render() {
+    return (
+      <div className="App">
+        <div className="App-body">
+          <Calculator />
+        </div>
+      </div>
+    );
+  }
+};
```
Comment (EricMbouwe) [inline src/App.js]: - [ ] Although this works, it is not necessary to use PureComponent here and assign the class to a const. This project requires the usage of a class-based component to be familiar with that approach of creating components in React.

- [ ] Here is an example of a class-based component. Let's try to follow the pattern
```
// MyComponent.js
import React, { Component } from 'react';

class MyComponent extends Component {
  render() {
    return (
      <div>This is my component.</div>
    );
  }
}

export default MyComponent;
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,270 @@
+import React, { PureComponent } from 'react';
+
+const Calculator = class extends PureComponent {
```
Comment (EricMbouwe) [inline src/components/Calculator.js]: - Same as above. Let's use a simple class-based component.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @codecaiine,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (codecaiine) [reply src/App.js]: Hey @EricMbouwe ,
Thank you for reviewing my work.
I completed all the requirements. 

Thank you.

**Review by AshakaE — APPROVED** [review]
### Hi @codecaiine  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**